globus_action_provider_tools/authentication.py

```diff
@@ -84,51 +79,20 @@ def introspect_token(self) -> GlobusHTTPResponse | None:
         resp = self.auth_client.oauth2_token_introspect(
             self.bearer_token, include="identity_set"
         )
-        now = time()
 
-        try:
-            if resp.get("active", False) is not True:
```

Comment:
I didn't like the old way that this was written (with `assert`), and even after the improvements I think we can do better. But it looks like this change removes these checks without any other location which examines `"active"`. Is that correct?

To my understanding, `active` is a response key which we must examine to be oauth2 compliant. It's possible that we need some of these other checks as well -- I've been meaning to look into that -- but that's the one which I'm pretty sure we need.

Maybe these are checked elsewhere and I've forgotten. Just LMK if I should look around.